src/styles/checkbox.css

```diff
@@ -1 +1,64 @@
 /* checkbox */
+
+/* default */
+.ui.checkbox label:before {
+  width: 1.5rem;
+  height: 1.5rem;
+  border: 1px solid var(--tertiary-cool-grey-900);
+  border-radius: 2px;
+}
+
+/* label */
+.ui.checkbox label {
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: var(--shade-black);
+  padding-left: var(--spacing-4);
+}
+
+/* checkmark */
+.ui.checkbox label:after {
+  color: var(--shade-white);
+  font-size: 1.25rem;
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+.ui.checkbox input:checked ~ label:after {
+  color: var(--shade-white);
+}
+
+/* focus */
+.ui.checkbox input:focus ~ label:before {
+  border-color: var(--primary-pms-362_900);
+}
+
+.ui.checkbox input:focus ~ label:after {
+  color: var(--shade-white) !important;
```

Comment:
Here I'm trying to override Semantic styles when checkbox just clicked. What happens is when I first click the checkmark is black, and then when I click somewhere else it changes to white.
I tried increasing class specificity by duplicating the class names, providing selectors in `:is()`.  None of which seem to work other than `!important`.
I'm happy to hear thoughts on how we can go about it.

Reply:
That's ok. Obviously let's try to avoid it, as you've done, but if we _need_ to use `!important`, then so be it. Thanks for trying other options.